### ISSUE TITLE
filtering visible nbextensions

### ISSUE THREAD

**Issue by juhasch** [Jupyter-contrib/jupyter_nbextensions_configurator#3]
I tried to install it on my Mac. 

The installation itself worked well, as well as enabling the server extension:
```
Jurgens-iMac:jupyter_nbextensions_configurator juhasch$ jupyter nbextensions_configurator enable --user
Enabling: jupyter_nbextensions_configurator
- Writing config: /Users/juhasch/.jupyter
    - Validating...
      jupyter_nbextensions_configurator  OK
```

However, the javascript files are not found. I tried copying them manually to the Jupyter `nbextensions` directory, but now I end up with:
```Javascript
main.js?v=20160620184014:1018 Uncaught (in promise) TypeError: Cannot read property 'length' of undefined(…)
build_extension_list @ main.js?v=20160620184014:1018
(anonymous function) @ main.js?v=20160620184014:959
```

Any ideas ?

**Comment by jcb91** [comment]
Oh dear :( I'm away for the weekend so can't check till I get back. The
idea is that the server extension edits the server's `nbextensions_path` to
insert the static path for the JavaScript - it should say something to this
effect in the server logs (may need to run with `--debug` flag to see it).
The main thing that springs to mind is that it may be an outdated template
somewhere?

**Comment by juhasch** [comment]
OK, my fault. I removed some old leftover stuff and now it works nicely!
I don't see the nbextensions tab on the main page, however. Maybe there is still something else left in my config.

**Comment by jcb91** [comment]
The dashboard tab isn't enabled by default, you have to enable it as a
separate nbextension from the regular interface. If you did that, then I'm
not sure what's precenting it from showing up - it does in the Travis is
builds, so perhaps it's another machine-specific leftovers issue?

**Comment by juhasch** [comment]
Found it. A case of RTFM...
I like this like it is a lot.

Now if we only did not have so many extensions. This gives so many choices :smirk:

Seriously, my extension list has nearly 100 entries. I guess we will need some kind of sorting or search in the future.

**Comment by jcb91** [comment]
> Found it. A case of RTFM

Perhaps, but I think having it enabled by default *is* pretty much the behaviour one would expect intuitively, so it might be worth making it so - I'm a fan of intuitive design making a manual unnecessary reading wherever possible.

> Seriously, my extension list has nearly 100 entries

Ahaha! Ok, yeah that could do with some filtering! I was thinking a while back that some kind of some kind of tagging system might be useful, but didn't get round to considering how that might actually work. Alternatively, something along the lines of the notebook find-and-replace using jquery typeahead (I think that's the name?)

**Comment by jcb91** [comment]
ok, PR enabling filtering at #17

**Comment by jcb91** [comment]
should be fixed by #17
